src/Phinx/Db/Adapter/AbstractAdapter.php

```diff
@@ -258,4 +263,44 @@ public function isDryRunEnabled()
 
         return ($input && $input->hasOption('dry-run')) ? (bool)$input->getOption('dry-run') : false;
     }
+
+    /**
+     * Adds user-created tables (e.g. not phinxlog) to a cached list
+     */
+    public function addCreatedTable($tableName)
+    {
+        if (substr_compare($tableName, 'phinxlog', -strlen('phinxlog')) !== 0) {
+            $this->createdTables[] = $tableName;
+        }
+    }
+
+    /**
+     * Updates the name of the cached table
+     */
+    public function updateCreatedTableName($tableName, $newTableName)
+    {
+        if (($key = array_search($tableName, $this->createdTables)) !== false) {
```

Comment:
It is usually cleaner to assign first, then go into the condition.

Reply:
Agreed, fixed.